mysql-exporter/mysql-exporter.libsonnet

```diff
@@ -1,40 +1,40 @@
 local k = import 'ksonnet-util/kausal.libsonnet';
 local container = k.core.v1.container;
-local service = k.core.v1.service;
 local containerPort = k.core.v1.containerPort;
 local deployment = k.apps.v1.deployment;
 
-{
+local secrets = import 'secrets.libsonnet';
+
+
+secrets {
+  image:: 'prom/mysqld-exporter:v0.12.1',
+
   _config:: {
     mysql_user: error 'must specify mysql user',
-    mysql_password: error 'must specify mysql password',
-    deployment_name: error 'must specify deployment name',
-    namespace: error 'must specify namespace',
+    mysql_password: '',
+    mysql_fqdn:: '%s.%s.svc.cluster.local' % [
```

Comment:
Are you aware that you moved `mysql_fqdn` into `_config`.

In the unlikely case that someone is using this lib, that would be an api change.

Reply:
Ah yes. I thought it was tidier, but there's a risk of breaking something. I'll move it back. Thanks!